#! /bin/sh /usr/share/dpatch/dpatch-run
## gpxlogger-gps.h.dpatch by Bernd Zeimetz <bernd@bzed.de>
##
## DP: fixing an implicit declaration

@DPATCH@
diff -urNad gpsd~/gpxlogger.c gpsd/gpxlogger.c
--- gpsd~/gpxlogger.c	2006-12-15 00:43:39.000000000 +0100
+++ gpsd/gpxlogger.c	2007-10-14 21:22:13.000000000 +0200
@@ -15,6 +15,8 @@
 #include <glib/gprintf.h>
 #include "gpsd_config.h"
 
+#include "gps.h"
+
 DBusConnection* connection;
 
 static char *author = "Amaury Jacquot";
